components/common/top-nav/TopNav.module.css

```
@@ -153,6 +153,40 @@ nav.community, .brokenLink, .brokenLink-learn, nav.rn {
 
+/* Announcement strip — same shape as the hackathon banner so it reads as
+   part of the site, in the Ballerina brand teal. */
+.announcementBanner {
+    text-align: center;
+    color: #fff;
+    padding: 10px 16px;
+    font-weight: 400;
+    font-size: 18px;
+    background: #20b6b0;
+    display: flex;
+    grid-gap: 16px;
```

Stylelint (17.14.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)